persistent/_timestamp.c

```diff
@@ -27,8 +27,15 @@ static char TimeStampModule_doc[] =
 "$Id$\n";
 
 
-
-#define SCONV ((double)60) / ((double)(0x10000)) / ((double)(0x10000))
+/* A magic constant having the value 0.000000013969839. When an
+   number of seconds between 0 and 59 is *divided* by this number, we get
+   a number between 0 (for 0), 71582786 (for 1) and 4223384393 (for 59),
+   all of which can be represented in a 32-bit unsigned integer, suitable
+   for packing into 4 bytes using `TS_PACK_UNSIGNED_INTO_BYTES`.
+   To get (close to) the original seconds back, use
+   `TS_UNPACK_UNSIGNED_FROM_BYTES` and *multiply* by this number.
+ */
+#define TS_SECOND_BYTES_BIAS ((double)60) / ((double)(0x10000)) / ((double)(0x10000))
```

Comment:
Thanks, this helps a bunch!

Reply:
I feel like I learned a lot in the process of documenting!

(Ooops, there's still an SCONV  in the Python implementation, I should rename that.)